Sources/KituraNet/HTTP/HTTPServerRequest.swift

```diff
@@ -24,9 +24,7 @@ import Foundation
 public class HTTPServerRequest: ServerRequest {
 
     /// Set of HTTP headers of the request.
-    public var headers : HeadersContainer {
-        return HeadersContainer.create(from: _headers)
-    }
+    public var headers : HeadersContainer
 
     private var _headers: HTTPHeaders
```

Comment:
Should we have `_headers` as well?

Reply:
The only place where we use is to get `Host`. Can we now re-use `headers.httpHeader()` to obtain the same if it is not expensive?

Reply:
@nethraravindran Yes, I agree. I've made the change.